app/frontend/src/components/RatingsSlider.tsx

```diff
@@ -0,0 +1,151 @@
+import Slider from "@material-ui/core/Slider";
+import { withStyles } from "@material-ui/core/styles";
+import MoodBadOutlinedIcon from "@material-ui/icons/MoodBadOutlined";
+import MoodOutlinedIcon from "@material-ui/icons/MoodOutlined";
+import SentimentDissatisfiedIcon from "@material-ui/icons/SentimentDissatisfied";
+import SentimentDissatisfiedOutlinedIcon from "@material-ui/icons/SentimentDissatisfiedOutlined";
+import SentimentSatisfiedIcon from "@material-ui/icons/SentimentSatisfied";
+import SentimentSatisfiedAltOutlinedIcon from "@material-ui/icons/SentimentSatisfiedAltOutlined";
+import SentimentVeryDissatisfiedOutlinedIcon from "@material-ui/icons/SentimentVeryDissatisfiedOutlined";
+import SentimentVerySatisfiedOutlinedIcon from "@material-ui/icons/SentimentVerySatisfiedOutlined";
+import React, { useRef } from "react";
+import { theme } from "theme";
+import makeStyles from "utils/makeStyles";
+
+const StyledSlider = withStyles({
```

Comment:
I thought this might be a little bit hack-y, because in order to do the customisation with makeStyles it has to be done like: 
```
const useStyles = makeStyles((theme) => ({
    root: {
      "&>.MuiSlider-thumb": {
        customisation
    }
```

to access the thumb, rail, etc. In [the documentation](https://material-ui.com/components/slider/#customized-sliders) customisation is done via a higher order component – do we still prefer `makeStyles` implementation?

Reply:
I don't mind that way, but you can also do it with normally named classes and then use the `classes` prop of the mui component with the same keys as you have used in `WithStyles` (in fact I think that's what it does - adds the object to that prop)

Reply:
You're right, somehow I missed the `classes` prop that allows you to do this.